Formula/proteinortho.rb

```diff
@@ -0,0 +1,22 @@
+class Proteinortho < Formula
+  desc "Detecting orthologous genes within different species"
+  homepage "https://gitlab.com/paulklemm_PHD/proteinortho"
+  url "https://gitlab.com/paulklemm_PHD/proteinortho/-/archive/v6.0/proteinortho-v6.0.tar.gz"
+  sha256 "e82522cdae62c182fc3bb75903e8ce3a388573fccbea16b72a0a8c47a653a34b"
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "lapack"
```

Comment:
@dpo Can OpenBLAS be used rather than LAPACK? Do you have a preference?

Reply:
i need the dsyevr/dsyevx function of lapack, i am not sure if that is also part of BLAS.

Reply:
We should move to OpenBLAS for new formulae, see #39226

Reply:
That's my understanding (but I'm not an expert).